app.py

```diff
@@ -34,6 +34,12 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def get_user_identity():
```

Comment:
Czy nie lepiej by tutaj było dać token jako parametr i podawać go z ciała funkcji która obsługuje zapytanie? Obawiam się że pobieranie w tym miejscu requesta jest lekko niebezpieczne (ale też może być spoko, nie wiem)

Reply:
Masz rację. Poprawie to :)